docs/serving/load-balancing/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Load balancing"
+weight: 30
+type: "docs"
+---
+
+You can configure load balancing on Knative, by placing the _Activator service_ in the request path to act as a load balancer.
+
+**NOTE:** To do this, you must first ensure that individual pod addressability is enabled.
```

Comment:
`For this to work, ...`
Since you can put Activator in the request path, just the load balancing will be whatever underlying transport is (mesh provider most likely).

Reply:
So do we actually need this note at all?
If a user needs to do something here maybe we should add that procedure or link to it to make it clear?